IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase+Internal.h

```diff
@@ -40,6 +42,17 @@
                                                  context:(id<MSIDRequestContext>_Nullable)context
                                                    error:(NSError*__nullable*__nullable)error;
 
++ (nullable NSString *)getPrimaryEccTenantWithSharedAccessGroup:(NSString *_Nullable)sharedAccessGroup
+                                                        context:(id <MSIDRequestContext> _Nullable)context
+                                                          error:(NSError *_Nullable *_Nullable)error;
+
+
++ (MSIDWPJMetadata *_Nullable)readWPJMetadataWithSharedAccessGroup:(NSString *_Nullable)sharedAccessGroup
```

Comment:
nit; Is it possible to keep the same format as `(nullable  MSIDWPJMetadata *)`?

Reply:
addressed